app/digitizingcontroller.cpp

```diff
@@ -141,11 +152,17 @@ void DigitizingController::onPositionChanged()
   if ( !mPositionKit->hasPosition() )
     return;
 
+
   QgsPoint point = mPositionKit->position();
   fixZ(&point);
 
-  mRecordedPoints.append( point );
-
+  if (mLastTimeRecorded.addSecs(mLineRecordingInterval) <= QDateTime().currentDateTime()) {
+      mLastTimeRecorded = QDateTime().currentDateTime();
```

Comment:
better to use QDateTime::currentTime() here and on the line above

Reply:
QTime::currentTime() ?